src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -83,8 +101,11 @@ export const toolUrl = appId => {
 
     case frontendApps.APPOINTMENTS:
       return {
-        url: `${mhvBaseUrl()}/mhv-portal-web/web/myhealthevet/scheduling-a-va-appointment`,
-        redirect: true,
+        url: [
+          `${mhvBaseUrl()}/mhv-portal-web/appointments`,
+          `${mhvBaseUrl()}/mhv-portal-web/web/myhealthevet/scheduling-a-va-appointment`,
+        ][index],
+        redirect: false,
```

Comment:
This and `serviceDescription` make use of the index passed from the initialization of the CTA widget via `querySelectorAll`, which iterates over all elements with the `cta-widget` class name.